python-avd/pyavd/_eos_designs/schema/schema_fragments/inband_ztp_bootstrap_file.schema.yml

```
@@ -0,0 +1,7 @@
+---
+type: dict
+keys:
+  inband_ztp_bootstrap_file:
+    type: str
```

@ClausHolbechArista I feel this one needs a description. I have not captured the manipulation for arista.io URL - any suggestion on how to word this properly?

```suggestion
    type: str
    description: |-
      bootstrap file configured in DHCP to use for inband ZTP.
      If not set and `cvp_instance_ips` is set then the bootstrap file value will be set to:
         
 `https://{cvp_instance_ips[0]}/ztp/bootstrap`
      Otherwise no value will be configured. 
      
```